### PR TITLE
Fix/health kit delete with UUID

### PR DESCRIPTION
**Pull request by alarm109**
- Fixed deletion with uuid in deleteRecord method.
- Added test for deleting records on HealthKit with date and uuid.

### CODE REVIEW

File: commitlint.config.js [hunk 1]
```diff
@@ -1 +1,6 @@
-module.exports = {extends: ['@commitlint/config-conventional']}
+module.exports = {
+  extends: ['@commitlint/config-conventional'],
+  rules: {
+    'scope-enum': [2, 'always', ['android', 'ios']],
```
Comment (lukebars) [inline commitlint.config.js:4]: Will this allow commits without scope?

File: example/src/sections/Delete.tsx [hunk 2]
```diff
@@ -0,0 +1,92 @@
...
+    HKWasUserEntered: 1,
+    logId: 8290,
+  },
+};
+
+export const Delete = () => {
+  const [record, setRecord] = useState<HealthDataRecord | undefined>(undefined);
+
+  const getHeartRateUUIDText = () => {
+    const text = record === undefined ? 'undefined' : record.uuid;
```
Comment (lukebars) [inline example/src/sections/Delete.tsx:25]: How about `record?.uuid ?? 'undefined' `

File: commitlint.config.js (hunk 1 above)
Comment (alarm109) [reply commitlint.config.js:4]: Yes, this allows default types without scopes and also these specific two scope.

File: example/src/sections/Delete.tsx (hunk 2 above)
Comment (alarm109) [reply example/src/sections/Delete.tsx:25]: For some reason this didn't work, it then returned blank to the string. But I will double check

**Review by lukebars — APPROVED** [review]
Looks good overall :shipit:

**Pull request merged**